<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XFPKPackerClass</class>
 <widget class="QMainWindow" name="XFPKPackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XFPKPacker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="TreeAsset">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>531</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGenerateFPX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>生成包</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditAssetDir">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOpen">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditFPKName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xfpkpacker.qrc"/>
 </resources>
 <connections/>
</ui>
